File: net/import-linked-content/ValidateContent.cs
// DocSection: import_linked_validate_content
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kentico.Kontent.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_PROJECT_ID>"
});

ManagementClient client = new ManagementClient(options);

var response = await client.ValidateProjectAsync();
// EndDocSection

File: js/import-linked-content/ValidateContent.js
// DocSection: import_linked_validate_content
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
// Using ES6 syntax
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  projectId: '<YOUR_PROJECT_ID>',
  apiKey: '<YOUR_API_KEY>'
});

const response = await client.validateProjectContent()
  .forProjectId('<YOUR_PROJECT_ID>')
  .toPromise();
// EndDocSection